lib/rubocop/cop/style/fetch_env_var.rb

```diff
@@ -69,9 +69,20 @@ def message_chained_with_dot?(node)
         # - Used as a flag (e.g., `if ENV['X']` or `!ENV['X']`) because
         #   it simply checks whether the variable is set.
         # - Receiving a message with dot syntax, e.g. `ENV['X'].nil?`.
-        # - `ENV['key']` is a receiver of `||=`, e.g. `ENV['X'] ||= y`.
+        # - `ENV['key']` assigned by  Logical AND/OR assignment.
```

Comment:
That's just my two cents :-)

```suggestion
        # - `ENV['key']` assigned by logical AND/OR assignment.
```

Reply:
Sorry for spending your time on my trivial mistake...💦